crates/bevy_window/src/window.rs

```diff
@@ -129,6 +131,7 @@ impl WindowResizeConstraints {
 }
 
 /// An operating system window that can present content and receive user input.
+/// To create a window, use a [`bevy_ecs::event::EventWriter`]`<`[`crate::CreateWindow`]`>`
```

Comment:
```suggestion
/// To create a window, use a [`EventWriter<CreateWindow>`]`(`crate::CreateWindow).
```
I think this is sufficiently clear to anyone who's managed to stumble across these docs, and should do less strange things to the formatting.